gwcs/utils.py

```diff
@@ -120,29 +120,30 @@ def _compute_lon_pole(skycoord, projection):
         lon = skycoord.spherical.lon.value
         lat = skycoord.spherical.lat.value
         unit = u.deg
-
     else:
         lon, lat = skycoord
         unit = None
         if isinstance(lon, u.Quantity):
-            lon = lon.to(u.deg).to_value()
+            lon = lon.to_value(u.deg)
             unit = u.deg
-
         if isinstance(lat, u.Quantity):
-            lat = lat.to(u.deg).to_value()
+            lat = lat.to_value(u.deg)
             unit = u.deg
 
-    cel = Celprm()
-    cel.ref = [lon, lat]
-    cel.prj.code = projection.prjprm.code
-    pvrange = projection.prjprm.pvrange
-    if pvrange:
-        i1 = pvrange // 100
-        i2 = i1 + (pvrange % 100) + 1
-        cel.prj.pv = i1 * [None] + list(projection.prjprm.pv[i1:i2])
-    cel.set()
-
-    lonpole = cel.ref[2]
+    if isinstance(projection, projections.Zenithal) and lat in (90, -90):
+        lonpole = 180
+    else:
+        cel = Celprm()
+        cel.ref = [lon, lat]
+        cel.prj.code = projection.prjprm.code
+        pvrange = projection.prjprm.pvrange
+        if pvrange:
+            i1 = pvrange // 100
+            i2 = i1 + (pvrange % 100) + 1
+            cel.prj.pv = i1 * [None] + list(projection.prjprm.pv[i1:i2])
+        cel.set()
+
+        lonpole = cel.ref[2]
```

Comment:
I wonder this change would be closer to the errata (default be phi_0 + 180):
```suggestion
    cel = Celprm()
    cel.ref = [lon, lat]
    cel.prj.code = projection.prjprm.code
    pvrange = projection.prjprm.pvrange
    if pvrange:
        i1 = pvrange // 100
        i2 = i1 + (pvrange % 100) + 1
        cel.prj.pv = i1 * [None] + list(projection.prjprm.pv[i1:i2])
    cel.set()
    if isinstance(projection, projections.Zenithal) and lat in (90, -90):
        lonpole = cel.prj.phi0 + 180
    else:
        lonpole = cel.ref[2]
```

Reply:
I am actually thinking of shortcutting the computation and setting `lon_pole=180` for all Zenithal projections (See Sec 2.2 in the paper II). Combined with the errata which defines it for theta_0 == +/- 90, this means `lon_pole` can be set to 180 for all Zenithal projections. Do you agree?

Reply:
Yes